pan3d/dataset_builder.py

```diff
@@ -37,6 +37,7 @@ def __init__(self, server=None, dataset_path=None, state=None, pangeo=False):
         server.client_type = "vue3"
         self.server = server
         self._layout = None
+        self._cloud = is_cloud_env()
```

Comment:
I'm not keen on that "cloud" name. My issue with that is that it does not reflect your intent. 
You are relying on an assumption that "cloud" means no GPU, and therefore, you should try to prevent server-side rendering. 

Instead, I would pick names like `self._force_local_rendering = not has_rendering_capabilities()` or something else but along those lines that explain your intent.

Reply:
Changed in [033505b](https://github.com/Kitware/pan3d/pull/48/commits/033505bd41f1a3067cf3c42cc846c271ea3baaa5)

Reply:
Please read my comment fully